internal/parser/simple_parser_rules.go

```diff
@@ -3642,6 +3660,11 @@ func (p *simpleParser) parseQualifiedTableName(r reporter) (stmt *ast.QualifiedT
 			r.unexpectedToken(token.KeywordIndexed)
 		}
 	}
+
+	if reflect.DeepEqual(stmt, &ast.QualifiedTableName{}) {
+		return nil
+	}
```

Comment:
Don't use reflect here, please.
```suggestion
	if (*stmt == ast.QualifiedTableName{}) {
		return nil
	}
```

Reply:
Ah I didnt know how to compare them that way. The brackets were the issue. Fixed.